.github/workflows/ci.yml

```diff
@@ -12,13 +12,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - run: npm ci
-      - uses: mattallty/jest-github-action@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Curious why this action needs `GITHUB_TOKEN`... seems strange!

Reply:
I think it uses the checks API to report results back to specific line numbers and such: https://github.com/mattallty/jest-github-action

Kinda cool but didn't seem to work too well.